sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -1,7408 +1,2542 @@
-            "calculateValue": "",
-            "calculateServer": false,
-            "widget": null,
-            "validateOn": "change",
-            "validate": {
-                "required": false,
-                "custom": "",
-                "customPrivate": false,
-                "strictDateValidation": false,
-                "multiple": false,
-                "unique": false
-            },
-            "allowCalculateOverride": false,
-            "encrypted": false,
-            "showCharCount": false,
-            "showWordCount": false,
-            "allowMultipleMasks": false,
-            "id": "ejox0w8"
-        },
-        {
-            "label": "Application Number | Location Name | Study Dates",
-            "tag": "p",
-            "className": "",
-            "attrs": [
-                {
-                    "attr": "",
-                    "value": ""
-                }
-            ],
-            "content": "{{ data.applicationNumber + \" | \" + data.institutionLocationName + \" | \" + data.studentStudyStartDate + \" - \" + data.studentStudyEndDate }}",
-            "refreshOnChange": true,
-            "customClass": "category-header-medium secondary-color mt-5 mb-5",
-            "hidden": false,
-            "modalEdit": false,
-            "key": "html17",
-            "tags": [],
-            "properties": {},
-            "conditional": {
-                "show": null,
-                "when": null,
-                "eq": "",
-                "json": ""
-            },
-            "customConditional": "",
-            "logic": [],
-            "attributes": {},
-            "overlay": {
-                "style": "",
-                "page": "",
-                "left": "",
-                "top": "",
-                "width": "",
-                "height": ""
-            },
-            "type": "htmlelement",
-            "input": false,
-            "tableView": false,
-            "placeholder": "",
-            "prefix": "",
-            "suffix": "",
-            "multiple": false,
-            "defaultValue": null,
-            "protected": false,
-            "unique": false,
-            "persistent": false,
-            "clearOnHide": true,
-            "refreshOn": "",
-            "redrawOn": "",
-            "dataGridLabel": false,
-            "labelPosition": "top",
-            "description": "",
-            "errorLabel": "",
-            "tooltip": "",
-            "hideLabel": false,
-            "tabindex": "",
-            "disabled": false,
-            "autofocus": false,
-            "dbIndex": false,
-            "customDefaultValue": "",
-            "calculateValue": "",
-            "calculateServer": false,
-            "widget": null,
-            "validateOn": "change",
-            "validate": {
-                "required": false,
-                "custom": "",
-                "customPrivate": false,
-                "strictDateValidation": false,
+    "forms": {
```

Comment:
please removed lines 1 and 2 and their respective closing brackets, we only need from line 3